src/explore-education-statistics-admin/src/pages/release/data/components/DataFilesTable.tsx

```diff
@@ -0,0 +1,239 @@
+          id,
+          label: title,
+        }))}
+        onCancel={() => {
+          setDataFiles(initialDataFiles);
+          onCancelReordering();
+        }}
+        onConfirm={() => onConfirmReordering(dataFiles)}
+        onMoveItem={({ prevIndex, nextIndex }) => {
+          const reordered = reorder(dataFiles, prevIndex, nextIndex);
+          setDataFiles(reordered);
+        }}
+        onReverse={() => {
+          setDataFiles(dataFiles.toReversed());
+        }}
+      />
+    );
+  }
+
+  return (
+    <table className={styles.table} data-testid="Data files table">
+      <thead>
+        <tr>
+          <th scope="col">Subject title</th>
+          <th scope="col">Size</th>
+          <th scope="col">Status</th>
+          <th scope="col">Actions</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {dataFiles.map(dataFile => (
+          <tr key={dataFile.title}>
+            <td data-testid="Subject title">{dataFile.title}</td>
+            <td data-testid="Data file size" className={styles.fileSize}>
+              {dataFile.fileSize.size.toLocaleString()} {dataFile.fileSize.unit}
+            </td>
+            <td data-testid="Status">
+              <ImporterStatus
+                className={styles.fileStatus}
+                releaseId={releaseId}
+                dataFile={dataFile}
+                onStatusChange={onStatusChange}
+              />
+            </td>
+            <td data-testid="Actions">
+              <ButtonGroup className={styles.actions}>
+                <Modal
+                  showClose
+                  title="Data file details"
+                  triggerButton={<ButtonText>View details</ButtonText>}
+                >
+                  <DataFileDetailsTable
+                    dataFile={dataFile}
+                    releaseId={releaseId}
+                    onStatusChange={onStatusChange}
+                  />
+                </Modal>
+                {canUpdateRelease &&
+                  terminalImportStatuses.includes(dataFile.status) && (
+                    <>
+                      {dataFile.status === 'COMPLETE' && (
+                        <>
+                          <Link
+                            to={generatePath<ReleaseDataFileRouteParams>(
+                              releaseDataFileRoute.path,
+                              {
+                                publicationId,
+                                releaseId,
+                                fileId: dataFile.id,
+                              },
+                            )}
+                          >
+                            Edit title
+                          </Link>
+                          {dataFile.publicApiDataSetId ? (
+                            <Modal
+                              showClose
+                              title="Cannot replace data"
+                              triggerButton={
+                                <ButtonText>Replace data</ButtonText>
```

Comment:
At smaller screens the word 'Replace' breaks across multiple lines and it looks quite weird:

![image](https://github.com/user-attachments/assets/e6b76d05-d83c-4219-9d8b-282976f4ecdd)